alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
With lint yes. You can still use [lintOptions](https://github.com/Tapadoo/Alerter/blob/sputnik/alerter/build.gradle#L29) to change the severity of rules. But with the rest I don't think so, all you can do with them is `@SuppressWarning` per method/class. But an update could be pushed to Sputnik allow it to read local rules in a similar style.